src/core_modules/capture-core/components/Pages/EditEvent/epics/editEvent.epics.js

```diff
@@ -15,7 +15,7 @@ import {
     orgUnitCouldNotBeRetrievedOnUrlUpdate,
     startOpenEventForEditInDataEntry,
 } from '../editEvent.actions';
-import { actionTypes as eventListActionTypes } from '../../MainPage/EventsList/eventsList.actions';
+import { listViewActionTypes as eventListActionTypes } from '../../../ListView';
```

Comment:
Repeating myself on this one. Maybe we can avoid the use of as here now that the action types have such a good distinct name ;)

Reply:
Removed in later PR.